src/axom/core/tests/core_memory_management.cpp

```diff
@@ -17,6 +17,10 @@
 // HELPER METHODS
 //------------------------------------------------------------------------------
 
+#ifdef AXOM_USE_UMPIRE
+#else
+#endif
```

Comment:
Why is this preprocessor `ifdef` block here?

Reply:
Oooops, missed that! My editor was folding pragmas in a weird way. Thanks for catching that. Fixed.